packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/richtext/richtext.component.ts

```diff
@@ -1,16 +1,55 @@
-﻿import { Component, OnInit } from '@angular/core';
+﻿import { Component, OnInit, ElementRef, Renderer2 } from '@angular/core';
 import { Field } from '@sitecore-jss/sitecore-jss-angular';
 import { SxaComponent } from '../sxa.component';
 
 @Component({
   selector: 'app-richtext',
   templateUrl: './richtext.component.html',
+  styles: [
+    `
+      :host(.footer-richtext) {
```

Comment:
Inside all of these styles I see a lot of "magic" numbers
e.g. "width: 17%", "nth-child(2)", "max-width: 992px"
to it means that apply kind of "hack" to this component, so the solution is not reliable

Reply:
maybe we can explore using attribute selectors for components instead element selectors?